mlrun/model_monitoring/db/tsdb/timescaledb/timescaledb_connector.py

```diff
@@ -20,6 +20,7 @@
 import mlrun
 import mlrun.common.schemas.model_monitoring as mm_schemas
 import mlrun.model_monitoring.db.tsdb.timescaledb.timescaledb_schema as timescaledb_schema
+from mlrun import config
```

Comment:
That's not how we import mlrun config though. Wouldn't expect it to work.

Reply:
I missed changes to tests/model_monitoring/test_writer_graph.py